Vagrantfile

```diff
@@ -77,6 +79,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box_version = ">= #{COREOS_VERSION}"
   config.vm.box_url = "#{upstream}/coreos_production_vagrant.json"
 
+  # Configure maven repository to use host folder
+  if MVN_SYNC_FOLDER != nil then
+    config.vm.synced_folder "#{MVN_SYNC_FOLDER}", "/home/core/.m2/repository", :nfs => true, :mount_options => ['nolock,vers=3,udp,noatime']    
+  end
+
```

Comment:
i'm not sure if this isn't going too, too, far feature specific. OTOH it would be nice to have a general way to achieve this in abstract for whatever use case, perhaps via some yaml file.